API/Parser/Parser.cs

```diff
@@ -320,6 +320,11 @@ public static class Parser
                 @"^(?<Series>.*)(?: |_)(?!\(\d{4}|\d{4}-\d{2}\))\(",
                 RegexOptions.IgnoreCase | RegexOptions.Compiled,
             RegexTimeout),
+            // spawn-123 (from https://github.com/Girbons/comics-downloader )
+            new Regex(
+                @"^(?<Series>.*)-(?<Chapter>\d*)",
```

Comment:
Try not to use .* as it can over consume characters. Get into the habit of using .+? instead which will consume as many as needed and not any more.